### DISCUSSION TITLE
Testing Qtile without removing my own qtile wm

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qtile/qtile#2700] [Q&A]
I want to start contributing but i don't see how i could test my Qtile branch without removing my own Qtile wm (cant have 2 of the same applications on the system ,can i ?) . The only thing i can think of is a VirtualEnv . Is there a proper way of doing this? 
I am very new to contributing to OS protects.

**Comment by elParaguayo** [comment]
I use a virtual environment on a separate TTY but that's just one way.

Have a look at this page of the docs: https://docs.qtile.org/en/latest/manual/hacking.html

There should be some good tips there.
